src/k8s/api/v1/cluster_config.go

```diff
@@ -77,13 +77,13 @@ type LocalStorageConfig struct {
 	Enabled       *bool   `json:"enabled,omitempty" yaml:"enabled,omitempty"`
 	LocalPath     *string `json:"local-path,omitempty" yaml:"local-path,omitempty"`
 	ReclaimPolicy *string `json:"reclaim-policy,omitempty" yaml:"reclaim-policy,omitempty"`
-	SetDefault    *bool   `json:"set-default,omitempty" yaml:"set-default,omitempty"`
+	Default       *bool   `json:"default,omitempty" yaml:"default,omitempty"`
 }
```

Comment:
Just to note: this is a breaking change in the API. We do not have stable releases yet, so we can do it.